opentimelineio/opentime.py

```diff
@@ -568,6 +568,69 @@ def to_timecode(time_obj, rate):
     )
 
 
+def from_time(time_str, rate):
```

Comment:
Could you rename this to `from_time_string` or `from_hhmmss` or something to make it clear what the input is expected to be?